snapcraft/README

```diff
@@ -0,0 +1,26 @@
+This folder contains the files needed to build a snap package for nikola.
+
+To build, in Ubuntu 16.04:
+
+apt install snapd snapcraft
+snap login some.account@whatever.com
+snapcraft
+cp nikola.py stage/usr/bin/nikola
+snapcraft
+
+To just use it:
```

Comment:
This should live in our getting started document when snaps actually work (i.e. you can build the demo site correctly). We should also mention other distros, and provide a link to snapcraft.io.